pp/integrations/plugin-packs/getting-started/how-to-guides/cma.md

```diff
@@ -0,0 +1,506 @@
+
+* Install the **centreon-monitoring-agent** package.
+
+1. Modify the **/etc/centreon-monitoring-agent/centagent.json** file (4 cases):
+
+<Tabs groupId="sync">
+<TabItem value="No encryption, agent connects to poller" label="No encryption, agent connects to poller">
+
+
+```json
+{
+    "log_level":"trace",
+    "endpoint":"<IP POLLER>:4317",
+    "host":"host_1",
+    "log_type":"file",
+    "log_file":"/var/log/centreon-monitoring-agent/centagent.log" 
+}
+```
+
+</TabItem>
+<TabItem value="Encryption, agent connects to poller" label="Encryption, agent connects to poller">
+
+```json
+{
+    "log_level":"trace",
+    "endpoint":"<IP POLLER>:4317",
+    "host":"host_1",
+    "log_type":"file",
+    "log_file":"/var/log/centreon-monitoring-agent/centagent.log" ,
+  "encryption":true,
+  "ca_certificate":"/tmp/ca_1234.crt"
+}
+```
+
+</TabItem>
+<TabItem value="No encryption, poller connects to agent" label="No encryption, poller connects to agent">
+
+```json
+{
+    "log_level":"trace",
+    "endpoint":"0.0.0.0:4317",
+    "host":"host_1",
+    "log_type":"file",
+    "log_file":"/var/log/centreon-monitoring-agent/centagent.log" ,
+  "reversed_grpc_streaming":true
+}
+```
+
+</TabItem>
+<TabItem value="Encryption, poller connects to agent" label="Encryption, poller connects to agent">
+
+```json
+{
+    "log_level":"trace",
+    "endpoint":"0.0.0.0:4317",
+    "host":"host_1",
+    "log_type":"file",
+    "log_file":"/var/log/centreon-monitoring-agent/centagent.log" ,
+  "reversed_grpc_streaming":true,
+  "encryption":true,
+  "private_key":"/tmp/server_1234.key",
+  "public_cert":"/tmp/server_1234.crt",
+  "ca_certificate":"/tmp/ca_1234.crt"
+}
+
+```
+
+</TabItem>
+</Tabs>
+
+#### Log configuration
+
+You can configure two kinds of log output:
+* file: the CMA logs into a file, the path is configured in the **log_file** option.
+* stdout: standard output is used.
+
+If you choose to log into a file, log rotation can be customized using the **log_max_file_size** and **log_max_files** options.
+
+Allowed log levels are: trace, debug, info, warning, error, critical and off.
+
+2. Restart the CMA:
+
+```shell
+systemctl restart centagent
+```
+
+</TabItem>
+<TabItem value="Windows" label="Windows">
+
+1. [Download the CMA installer](https://github.com/centreon/centreon-collect/releases) on every server you want to monitor.
+
+2. Start the installer (during configuration, you can click on (i) icons to have some help)
+   
+3. Configure endpoint and connection
+   You have to set "Host name in Centreon". This name must be the host name you will enter in Centreon UI.
+   In most case (agent connects to poller), you have to enter Poller endpoint. It must be like  "poller ip or dns name":"opentelemetry listening poller port usually 4317", for example 192.168.45.32:4317. 
+   In Poller-initiated connection (poller connects to agent), you have to choose an host interface or 0.0.0.0 and the listening port, usually 4317 on witch poller will connect.
```

Comment:
```suggestion
   * In case of a poller-initiated connection (the poller connects to the agent), you have to choose a host interface (0.0.0.0 for all interfaces) and the listening port on which poller will connect, usually 4317.
```